C# Star Practices/SimpleCalculator/SimpleCalculator.Tests/Calculator.Base.Test.cs

```diff
@@ -0,0 +1,101 @@
+namespace SimpleCalculator.Tests;
+
+using SimpleCalculator.Business;
+using SimpleCalculator.Business.Enums;
+using SimpleCalculator.Business.Abstraction;
+using SimpleCalculator.Business.OperatorBusiness;
+using SimpleCalculator.Business.OperatorBusiness.Operators;
+
+// test Calculator object, mocking the provider to make sure it's working as it should, and test it somewhere else
+
+public class CalculatorSumTest
+{
+    private readonly Calculator _calculator;
+
+    public CalculatorSumTest()
+    {
+        var mockedOperatorProvider = new Mock<IOperatorProvider>();
+        mockedOperatorProvider.Setup(x => x.GetOperator(OperatorEnum.sum)).Returns(new SumOperator());
+        this._calculator = new Calculator(mockedOperatorProvider.Object);
+    }
+
+    [Theory]
+    [InlineData(1, 2, 3)]
+    [InlineData(2, 2, 4)]
+    [InlineData(3, -2, 1)]
+    public void Add_TwoNumbers_ReturnsSum(int firstNumber, int secondNumber, int expected)
```

Comment:
the name of the method implies that it takes two inputs and returns their sum. But in reality the method has no return value and only compares the sum of the two inputs with the expected value. Try using the naming conventions of unit testing.